tests/integration/remmenders/dataset/test_criteo.py

```diff
@@ -4,7 +4,7 @@
 import os
```

Comment:
@miguelgfierro I think the test folder `integraton/remmenders` is misspelled 😄

Reply:
good catch, thanks!
It's fixed now